### ISSUE TITLE
Set cursor focus to input fields

### ISSUE THREAD

**Issue by jtrott** [authelia/authelia#511]
When the login page first loads, you can't start typing your username until you click with your mouse on the username field.
Simalarly, on the TOTP input page, you have to click with your mouse on the first digit box before you can start typing.
This slows down the workflow of logging in, it would be much better to have the focus set automatically so users can just start typing, then tab to the password field, then enter to login.

**Comment by clems4ever** [comment]
Hello @jtrott , thank you for reporting that. I think it's easy to fix. I will do it shortly.

**Comment by nightah** [comment]
Duplicate of #367.

**Comment by clems4ever** [comment]
Reopening this issue and closing #367 instead as it was reported for v3.

**Comment by clems4ever** [comment]
I had a look yesterday and it's not a trivial fix since the `autoFocus` property on TextField only applies when mounting a component for the first time.

Unfortunately, due to Authelia routing to the correct stage according to current level of privileges of the user, the component is mounted but rendered multiple times making the autoFocus feature not working.

It will require a bit of refactoring to fix.
